lib/screens/dup_screen_data/special_cases.ex

```diff
@@ -0,0 +1,337 @@
+defmodule Screens.DupScreenData.SpecialCases do
+  @moduledoc false
+
+  alias Screens.Config.Dup
+
+  alias Screens.Config.Dup.Override.{
+    FreeTextLine,
+    PartialAlert,
+    PartialAlertList
+  }
+
+  alias Screens.DupScreenData.{Data, Request}
+
+  def handle_special_cases(
+        %Dup.Departures{
+          sections: [
+            %Dup.Section{
+              stop_ids: ["place-kencl"] = stop_ids,
+              route_ids: ["Green-B", "Green-C", "Green-D"] = route_ids,
+              pill: pill
+            }
+          ]
+        } = primary_departures,
+        rotation_index
+      ) do
+    alerts = Request.fetch_alerts(stop_ids, route_ids)
+
+    interpreted_alerts =
+      alerts
+      |> Enum.map(fn alert ->
+        Map.put(
+          Data.interpret_alert(alert, stop_ids, pill),
+          :routes,
+          Data.alert_routes_at_station(alert, stop_ids)
```

Comment:
Just to make this slightly clearer could you use pipes? `alert |> Data.interpret_alert(...) |> Map.put(...)`